src/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <>
+      <div className="App">
+        <h1 className="title">Calculator</h1>
+        <Display />
+        <ButtonPanel />
+      </div>
+    </>
```

Comment:
@Alexoid1, good job trying to make the necessary correction.

**According to the previous reviewer:** ` App.jsx has children components wrapped in React.Fragment`

**Reason for using `React.Fragment`:** We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.

Example: 
```
    <>
        <h1 className="title">Calculator</h1>
        <Display />
        <ButtonPanel />
    </>
```

Reply:
please https://es.reactjs.org/docs/fragments.html#short-syntax
Read the documentation please <></> that the short syntax of a react.fragment

Reply:
@Alexoid1,

You need to remove the extra `div` you added, kindly read the reason for using `React.Fragment` above.

Do you get the whole idea now?